### PR TITLE
update copy post copy guidelines review

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/392891325557410/1194008532610029
Tech Design URL:
CC:

**Description**:

Update copy after copy guidelines review.  Copy has been checked and product review approved.

Note: Translations pending.

**Steps to test this PR**:
1. Launch the app and do a broad smoke test.

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -239,10 +239,10 @@
 "donotsell.info.headertext" = "Your data shouldn't be for sale. At DuckDuckGo, we agree. Activate the \"Global Privacy Control\" (GPC) setting and we'll signal to websites your preference to:\n\n• Not sell your personal data.\n• Limit sharing of your personal data to other companies.";
```
Comment (bwaresiak) [inline DuckDuckGo/en.lproj/Localizable.strings]: Seems like GPC strings use wrong apostrophes: `'`

File: adhocExportOptions.plist [hunk 2]
```diff
@@ -7,6 +7,27 @@
...
+
+        <key>com.duckduckgo.mobile.ios.QuickActionsTodayExtension</key>
+        <string>iOS 14 - Ad Hoc - Quick Actions</string>
+
+        <key>com.duckduckgo.mobile.ios.OpenAction2</key>
+        <string>iOS 14 - Ad Hoc - Open Action</string>
+
+        <key>com.duckduckgo.mobile.ios.Widgets</key>
+        <string>iOS 14 - Ad Hoc - Widgets</string>
+	</dict>
```
Comment (bwaresiak) [inline adhocExportOptions.plist]: Nice 👍

**Review by bwaresiak — APPROVED** [review]
Great job, gave this a smoke test and went through the code, apart from one apostrophe thingy (not even related to your changes) this looks good to me 👍

**Comment by brindy** [comment]
Thanks for the help @bwaresiak !

**Pull request merged**